jaxrs-api/src/main/java/se/fortnox/reactivewizard/jaxrs/Stream.java

```diff
@@ -11,5 +11,18 @@
 @Target({ElementType.METHOD})
 @Retention(RetentionPolicy.RUNTIME)
 public @interface Stream {
+    /**
+     * Stream format. If it set to CONCATENATED_JSON_OBJECTS,
+     * then the format will be just like an Observable stream
+     * resource method. If it is set to JSON_ARRAY then it will
+     * wrap the stream in a JSON array. This option applies only
+     * for the resource methods returning Flux.
+     * @return a stream type
+     */
+    Type value() default Type.JSON_ARRAY;
 
+    enum Type {
+        CONCATENATED_JSON_OBJECTS,
```

Comment:
Can you somehow mark concatenated_json_objects as deprected?

Reply:
Why are we _adding_ an option that we mark as deprecated...?

Reply:
For allowing backward compatibility. Some services are currently producing the "concatenated JSON format" and converting the endpoint to reactor will change the format, resulting in breaking changes. adding concatenated_json_object to the endpoint stream annotation will allow clients to consume the data as usual. Is just a way to mark the old way of streaming data.

If the concatenated json shall be deprecated may of course be discussed I think. 

Would you rather see it as a nondeprecated format?

Reply:
Ok, thanks for explaining :-)